unity3d/Assets/Swrve/SwrveSDK/SwrveLog.cs

```diff
@@ -10,18 +10,25 @@ public static class SwrveLog
     /// Set to false to disable the logging produced
```

Comment:
I put this one on hold while you were away. Made a few local changes but nothing crazy. Will get back to it and get onto you soon about what alternative options we have.